src/main/CommandHandler.cpp

```diff
@@ -291,6 +291,8 @@ CommandHandler::peers(std::string const& params, std::string& retStr)
                 peerNode["olver"] = (int)peer.second->getRemoteOverlayVersion();
                 peerNode["id"] =
                     mApp.getConfig().toStrKey(peer.first, fullKeys);
+                peerNode["flow_control"] =
```

Comment:
Done. I also added the number of flow-controlled connections to the `info` endpoint.

Reply:
Thanks! I looked at the overall size of the "info" endpoint, and I don't think we should add that one there: we try to keep it not too large so that one can see its output in a standard console (so every line counts).

Reply:
I think what we discussed offline last week was adding a metric so that we can track the relative number of connections with flow control enabled compared to the total number of connections (`overlay.connection.authenticated`)

Reply:
ah ok, I updated it